src/Symfony/Component/Cache/Marshaller/DefaultMarshaller.php

```diff
@@ -21,17 +21,17 @@
 class DefaultMarshaller implements MarshallerInterface
 {
     private bool $useIgbinarySerialize = true;
-    private bool $throwOnSerializationFailure = false;
 
-    public function __construct(bool $useIgbinarySerialize = null, bool $throwOnSerializationFailure = false)
-    {
+    public function __construct(
+        bool $useIgbinarySerialize = null,
+        private bool $throwOnSerializationFailure = false,
```

Comment:
Should be reverted, property has a default value

Reply:
Ah you mean it has a default value on the property, right?

Reply:
Yes :)